test/files/pos/CharSequence-implicits.scala

```diff
@@ -0,0 +1,4 @@
+object Test extends App {
+  Array('a', 'b'): CharSequence
+  IndexedSeq('a', 'b'): CharSequence
+}
```

Comment:
Since it's a library fix, it could be a junit test, just not in `scala` package.